klippy/extras/extruder_stepper.py

```diff
@@ -0,0 +1,25 @@
+# Code for supporting multiple steppers in single filament extruder.
+#
+# Copyright (C) 2019-2020  Simo Apell <simo.apell@live.fi>
```

Comment:
I don't think we should copyright into the future, so I wouldn't list 2020 as the year.

Reply:
Thanks for checking and for the comments. Wasn't aware how to apply copyright notes properly, will update that.